src/common/CONSTANTS.js

```diff
@@ -2,7 +2,7 @@
 
 const CHROMECAST_RECEIVER_APP_ID = '1634F54B';
 const SUBTITLES_SIZES = [75, 100, 125, 150, 175, 200, 250];
-const SUBTITLES_FONTS = ['Roboto', 'Arial', 'Halvetica', 'Times New Roman', 'Verdana', 'Courier', 'Lucida Console', 'sans-serif', 'serif', 'monospace'];
+const SUBTITLES_FONTS = ['Roboto Flex', 'Roboto', 'Arial', 'Halvetica', 'Times New Roman', 'Verdana', 'Courier', 'Lucida Console', 'sans-serif', 'serif', 'monospace'];
```

Comment:
Should we keep 'Roboto' here or replace it with 'Roboto Flex' instead?

Reply:
In my opinion we should keep 'Roboto' as a fallback option.